packages/react/src/views/Message/MessageHeader.js

```diff
@@ -72,6 +73,16 @@ const MessageHeader = ({
         return '';
     }
   };
+  const setExclusiveState = useSetExclusiveState();
+  const { setShowCurrentUserInfo, setCurrentUser } = useUserStore((state) => ({
+    setShowCurrentUserInfo: state.setShowCurrentUserInfo,
+    setCurrentUser: state.setCurrentUser,
+  }));
+
+  const handleAvatarClick = () => {
```

Comment:
Can you please change the function name ? It should be handleUsernameClick or something like that